app/controllers/employees/list.js

```diff
@@ -0,0 +1,12 @@
+import { db } from 'app/utils/db.server';
+
+const listEmployees = async (id) => {
+  const employees = await db.$queryRaw`
```

Comment:
Are we using some ORM? @GeorginaFraire @jackbravo ?

Reply:
Yes, we use prisma.

Reply:
@Carlos-Reyna So, I think we can use Prisma to do this query easier to read through of our ORM.

https://www.prisma.io/docs/concepts/components/prisma-client/relation-queries